casper/src/main/scala/coop/rchain/casper/MultiParentCasperImpl.scala

```diff
@@ -299,7 +299,13 @@ class MultiParentCasperImpl[F[_]
         _ <- EitherT(
               EquivocationDetector.checkNeglectedEquivocationsWithUpdate(b, s.dag, approvedBlock)
             )
-        _      <- EitherT.liftF(Span[F].mark("neglected-equivocation-validated"))
+        _ <- EitherT.liftF(Span[F].mark("neglected-equivocation-validated"))
+
+        // This validation is only to punish validator which accepted lower price deploys.
+        // And this scan happen if not configured correctly.
```

Comment:
It's a small typo here, you meant _can happen_ right?

Reply:
I just copied your [comment](https://github.com/rchain/rchain/pull/3556#discussion_r765490942) as explanation :relaxed:

Reply:
Yes, sorry my typo.  :) I corrected it in the comment.

Reply:
It's Ok :) I fixed typo in code comment